tests/templates/kuttl/external-access/install-airflow-cluster.yaml.j2

```diff
@@ -21,22 +21,28 @@ metadata:
   name: airflow
 spec:
   image:
-{% if test_scenario['values']['airflow'].find(",") > 0 %}
-    custom: "{{ test_scenario['values']['airflow'].split(',')[1] }}"
-    productVersion: "{{ test_scenario['values']['airflow'].split(',')[0] }}"
+{% if test_scenario['values']['airflow-latest'].find(",") > 0 %}
+    custom: "{{ test_scenario['values']['airflow-latest'].split(',')[1] }}"
+    productVersion: "{{ test_scenario['values']['airflow-latest'].split(',')[0] }}"
```

Comment:
Is it on purpose to only test this (and others) for 3.x?

Reply:
that one should probably be for all airflow versions, yes, as it is new functionality.

Reply:
Changed here: [8d65054](https://github.com/stackabletech/airflow-operator/pull/630/commits/8d65054d92a6a5578adaff4d2e75d2008d278757)